common/predictive-text/unit_tests/in_browser/cases/top-level-lmlayer.spec.ts

```diff
@@ -1,30 +1,35 @@
-import { assert } from '../../../../../node_modules/chai/chai.js';
+import { assert } from 'chai';
 
-import { LMLayer, Worker as WorkerBuilder }   from "../../../build/lib/web/index.mjs";
-import * as helpers from "../helpers.mjs";
+import { LMLayer, Worker as WorkerBuilder }  from "@keymanapp/lexical-model-layer/web";
+import { defaultCapabilities } from '../helpers.mjs';
 
 describe('LMLayer', function () {
-  this.timeout(testconfig.timeouts.standard);
+  this.timeout(5000);
```

Comment:
Do we need a new config file to replace this test constant? I see it 3x on this PR

Reply:
I'll mark it as a TODO / issue to perform after the full set of test conversions, as I kind of did the same thing across nearly all of the related PRs.  A one-shot cleanup PR would likely suffice.